app/code/Magento/CustomerGraphQl/Model/Context/AddUserInfoToContext.php

```diff
@@ -0,0 +1,80 @@
+<?php
```

Comment:
Extension attributes must be declared in `extension_attributes.xml`

Reply:
Fixed